cmd/issueShow.go

```diff
@@ -0,0 +1,100 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/xanzy/go-gitlab"
+	"github.com/zaquestion/lab/internal/git"
+	lab "github.com/zaquestion/lab/internal/gitlab"
+)
+
+var issueShowCmd = &cobra.Command{
+	Use:        "show [remote]",
+	ArgAliases: []string{"s", "d"},
+	Short:      "describe an issue",
+	Long:       ``,
+	Run: func(cmd *cobra.Command, args []string) {
+		issueNum := args[0]
+		if len(args) > 1 {
+			issueNum = args[1]
+		}
+		issueNumber, err := strconv.Atoi(issueNum)
+		if err != nil {
+			log.Fatal(err)
+		}
+		remote, _, err := parseArgsRemote(args)
+		if err != nil {
+			log.Fatal(err)
+		}
+		if remote == "" {
+			remote = forkedFromRemote
+		}
+		rn, err := git.PathWithNameSpace(remote)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		issue, err := lab.IssueGet(rn, issueNumber)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		printIssue(issue, rn)
+	},
+}
+
+func printIssue(issue *gitlab.Issue, project string) {
+	milestone := "None"
+	timestats := "None"
+	dueDate := "None"
+	assignee := "None"
+	state := map[string]string{
+		"opened": "Open",
+		"closed": "Closed",
+	}[issue.State]
+	if issue.Milestone != nil {
+		milestone = issue.Milestone.Title
+	}
+	if issue.TimeStats != nil && issue.TimeStats.HumanTimeEstimate != "" &&
+		issue.TimeStats.HumanTotalTimeSpent != "" {
+		timestats = fmt.Sprintf(
+			"Estimated %s, Spent %s",
+			issue.TimeStats.HumanTimeEstimate,
+			issue.TimeStats.HumanTotalTimeSpent)
+	}
+	if issue.DueDate != nil {
+		dueDate = time.Time(*issue.DueDate).String()
+	}
+	if issue.Assignee.Username != "" {
+		assignee = issue.Assignee.Username
+	}
+
+	fmt.Printf(`
```

Comment:
This should be a template, but I'm tempted to say we can follow up on it.
Sample: https://github.com/zaquestion/lab/blob/master/cmd/issueCreate.go#L71